use_helper_frame: true # CRITICAL: In the current setup, cam4_sensor_frame_helper should be used.
global_frame: map
pose_frame: base
memory_checking_period: 5 # seconds
max_meshing_height: 4 # meters
memory_purge_threshold: 1000 # in MB,
output_dir: "" # By default saves to "home/<username>/maps" folder
voxel_size: 0.08 #0.05
slice_height: 0.75
min_height: 0.05
max_height: 2
esdf: false
esdf_2d: false
mesh: true
distance_slice: false
tsdf_integrator_max_integration_distance_m: 30.0
tsdf_integrator_truncation_distance_vox: 4.0 # when increase it populates faster but so noisy
tsdf_integrator_max_weight: 800.0 #higher is better for high quality
color_integrator_max_integration_distance_m: 30.0
esdf_integrator_min_weight: 2.0
esdf_integrator_max_distance_m: 10.0
esdf_integrator_min_site_distance_vox: 1.0
max_tsdf_update_hz: 0.0
max_color_update_hz: 0.0
max_mesh_update_hz: 0.0
max_esdf_update_hz: 2.0
mesh_integrator_min_weight: 0.00001
mesh_integrator_weld_vertices: false
use_tf_transforms: true
use_topic_transforms: false
subscribers:
  depth_image_1_topic: "/point_cloud_colorizer_ros/depth_image_camera_1"
  depth_image_2_topic: "/point_cloud_colorizer_ros/depth_image_camera_2"
  depth_image_3_topic: "/point_cloud_colorizer_ros/depth_image_camera_3"
  rgb_image_1_topic: "/alphasense_driver_ros/cam3/dropped/debayered/slow"
  rgb_image_2_topic: "/alphasense_driver_ros/cam4/dropped/debayered/slow"
  rgb_image_3_topic: "/alphasense_driver_ros/cam5/dropped/debayered/slow"
  camera_info_1_topic: "/camera_utils/alphasense_cam3/cameraInfo"
  camera_info_2_topic: "/camera_utils/alphasense_cam4/cameraInfo"
  camera_info_3_topic: "/camera_utils/alphasense_cam5/cameraInfo"